### PR TITLE
Add Arbitrum docs

### PR DESCRIPTION
**Pull request by akegaviar**
Also fixed relnotes.

### CODE REVIEW

File: docs/platform/join-a-public-network.md [hunk 1]
```diff
@@ -67,6 +67,19 @@ The node status will change from **Pending** to **Running** once deployed.
 
 The node status will change from **Pending** to **Running** once deployed.
 
+## Join an Arbitrum network
+
+1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
+1. Under **Blockchain protocol**, select **Arbitrum**.
+1. Under **Blockchain network**, select **One mainnet** or **Goerli testnet**. Click **Next**.
+1. Under **Type**, select whether to run an [elastic](/glossary/elastic-node) or a [dedicated](/glossary/dedicated-node) node.
+1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/avalanche/modes).
```
Comment (PekopT) [inline docs/platform/join-a-public-network.md:76]:
```suggestion
1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/arbitrum/modes).
```

File: docs/platform/supported-protocols.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ Public network:
 * [Avalanche](/blockchains/avalanche)
 	* Mainnet. [Full and archive nodes](/operations/avalanche/modes).
 	* Fuji testnet
+* [Arbitrum](/blockchains/arbitrum)
+	* Arbitrum One mainnet. [Full and archive nodes](/operations/arbitrum/modes).
+	* Arbitrum Goerlu testnet
```
Comment (PekopT) [inline docs/platform/supported-protocols.md:29]:
```suggestion
	* Arbitrum Goerli testnet
```

**Review by PekopT — CHANGES_REQUESTED** [review]
@akegaviar

**Review by PekopT — CHANGES_REQUESTED** [review]
let's fix 2 points

**Comment by akegaviar** [comment]
@PekopT fixed, thank you very much for fixing.

File: docs/operations/arbitrum/tools.md [hunk 3]
```diff
@@ -0,0 +1,701 @@
+---
+meta:
+  - name: description
+    content: Learn how to interact with your Arbitrum node, deploy smart contracts through your node, and develop dapps.
+  - name: keywords
+    content: arbitrum web3 dapp geth ntitro one tooling
```
Comment (PekopT) [inline docs/operations/arbitrum/tools.md:6]:
```suggestion
    content: arbitrum web3 dapp geth nitro one tooling
```

File: docs/operations/arbitrum/tools.md [hunk 4]
```diff
@@ -0,0 +1,701 @@
...
+---
+
+# Tools
+
+## Interaction tools
+
+### MetaMask
+
+You can set your [MetaMask](https://metamask.io/) to interact through your Abitrum nodes deployed with Chainstack.
```
Comment (PekopT) [inline docs/operations/arbitrum/tools.md:15]:
```suggestion
You can set your [MetaMask](https://metamask.io/) to interact through your Arbitrum nodes deployed with Chainstack.
```

**Review by PekopT — APPROVED** [review]
LGTM

**Pull request merged**